src/main.rs

```diff
@@ -138,30 +137,50 @@ fn main() -> anyhow::Result<()> {
     setup_logging(args.verbose);
 
     // Default to current invocation path.
-    let path = args
+    let paths = args
         .path
-        .unwrap_or_else(|| env::current_dir().expect("Failed to get current directory"));
+        .unwrap_or_else(|| vec![env::current_dir().expect("Failed to get current directory")]);
 
     if let Criterion::Stamp = criterion {
-        debug!("Writing timestamp file in: {:?}", path);
+        if paths.len() > 1 {
+            anyhow::bail!("Using multiple paths and --stamp is currently unsupported")
+        }
+
+        debug!("Writing timestamp file in: {:?}", paths[0]);
         return Timestamp::new()
-            .store(path.as_path())
+            .store(paths[0].as_path())
             .context("Failed to write timestamp file");
-    }
+    };
+    // FIXME: Change to write to every passed in path instead of just the first one
+    // if let Criterion::Stamp = criterion {
+    //     return paths.iter().try_for_each(|path| {
+    //         debug!("Writing timestamp file in: {:?}", path);
+    //         Timestamp::new()
+    //             .store(path.as_path())
+    //             .context("Failed to write timestamp file")
+    //     });
+    // }
```

Comment:
If you want to keep this FIXME, please move it to line 144, but still delete this dead code.
```suggestion
```

Reply:
Kept the FIXME, removed the deadcode